scripts/NonEvmPeers.json

```diff
@@ -8,6 +8,14 @@
             "fxs": "0x4939035f8dd13d15a9386e28b6705519aa6f488791323466a3c0116a201e51aa",
             "sfrxUSD": "0x8602f005ca65b6da46a3c6ce66ecd1d15be911ca650d5f418d369df184b584cf",
             "sFrxEth": "0xbf2f1fc27286a43f25b05bd843a74a5478c4246343fa90c1fcb641a1caf46c61"
+        },
```

Comment:
Should movement peers be added here?

Reply:
assets at index 1 are addresses for aptos/movement